CityActivitas/city-activitas-backend
# Apply UV as dep management tool and add a basic test case

* build(uv): add postgrest
* test: add minimum test cases
* style: fix style through ruff
* test: add minimum health check test case
* test: init test client
* build(docker): Use uv to generate venv in docker
* style: fix style thorugh ruff and fix import path
* build(uv): simplify dep to only required packages
* build: initialize dev tools through Lee-W/cookiecutter-python-template
